.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: node_js
 node_js:
-  - "0.12"
+  - 6
+  - 4
```

Comment:
Having two versions specified here means that the tests will run *twice*, which seems completely unnecessary to me. (And I'm assuming that's what https://github.com/mozilla/pdf.js/pull/7500#discussion_r71947276 refers to.)

Hence I'd suggest that you simply remove this line, and please squash the commits afterwards (see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits).